x-pack/solutions/security/packages/kbn-cloud-security-posture/graph/src/components/graph/graph.tsx

```diff
@@ -182,7 +200,7 @@ export const Graph = memo<GraphProps>(
 
 Graph.displayName = 'Graph';
 
-const processGraph = (
+export const processGraph = (
```

Comment:
nit: instead of exposing an internal piece of code there's an `extractEdges` utility you can use for storybook testing.

it is a helper function that keeps the mock data readable and process it later for the graph